src/bundle.rs

```diff
@@ -298,12 +298,15 @@ impl SpendDescriptionV5 {
         Self { cv, nullifier, rk }
     }
 
-    pub fn into_spend_description(
+    pub fn into_spend_description<A>(
```

Comment:
This feels like it should be backwards-compatible, but I'll check with `cargo semver-checks`.

Reply:
#144 is not backwards-compatible, and that's the API change I actually needed, so I'm just going to deploy this at the same time.